gradle.properties

```diff
@@ -21,6 +21,6 @@ POM_DEVELOPER_ID=cashapp
 POM_DEVELOPER_NAME=CashApp
 POM_DEVELOPER_URL=https://github.com/cashapp/
 
-kotlin.js.compiler=both
+kotlin.js.compiler=ir
```

Comment:
FYI. Legacy JS cannot do timeouts in coroutines running on different dispatcher. Wild!